docs/index.md

```diff
@@ -2,6 +2,20 @@
     <img src="assets\images\build-with-encryption.jpg">
 </p>
 
-Welcome to Obscuro. Obscuro hyper-scales and encrypts Ethereum. No SDKs, 100% EVM.
+Welcome to Obscuro - the first Ethereum L2 with private smart contract state. 
 
-On this docsite you will find useful guidance on Obscuro, how to participate in the Testnet and, if you want to go deeper, read the whitepaper using the menu above. A PDF version of the whitepaper is [available](https://whitepaper.obscu.ro/assets/images/obscuro-whitepaper-0-10-0.pdf).
+Obscuro hyper-scales and encrypts Ethereum. No SDKs, 100% EVM.
```

Comment:
Maybe don't mention "no SDKS", for some reason it triggers people (like at the workshop).
Either way, add 100% Solidity too as this is one of our key USPs (no new languages to learn)